src/nvim/syntax.c

```diff
@@ -5781,6 +5781,7 @@ static char *(highlight_init_both[]) =
       "StatusLine term=reverse,bold cterm=reverse,bold gui=reverse,bold"),
   CENT("StatusLineNC term=reverse cterm=reverse",
       "StatusLineNC term=reverse cterm=reverse gui=reverse"),
+  "default link EndOfBuffer NonText",
```

Comment:
It was like this in the original patch, it has the same indent as the CENT items, which are at the same structural level.